rskj-core/src/main/java/co/rsk/core/bc/BlockExecutor.java

```diff
@@ -625,6 +625,7 @@ private BlockResult executeForMiningAfterRSKIP144(
                 }
 
                 loggingDiscardedBlock(block, tx);
+                txindex++;
```

Comment:
shouldn't we increment txindex also at line 644?

Reply:
Ohh true!